### PR TITLE
RAV-1200 : Handling for ascii method

### PR DESCRIPTION
**Pull request by vaibhavjain-dm**
RAV-1200 : Handling for ascii method

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java [hunk 1]
```diff
@@ -145,6 +145,8 @@
   /** Converts string_expr to a NUMBER data type. */
   public static final SqlFunction TO_NUMBER = SqlLibraryOperators.TO_NUMBER;
 
+  public static final SqlFunction IF = SqlLibraryOperators.IF;
```
Comment (soma-mondal) [inline core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java]: please remove it

File: core/src/main/java/org/apache/calcite/sql/SqlKind.java [hunk 2]
```diff
@@ -780,6 +780,16 @@
...
 
+  /**
+   * The "ASCII" function.
+   */
+  ASCII,
+
+  /**
+   * The "IF" function.
+   */
+  IF,
```
Comment (soma-mondal) [inline core/src/main/java/org/apache/calcite/sql/SqlKind.java]: please use other_function

File: core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java (hunk 1 above)
Comment (vaibhavjain-dm) [reply core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java]: Done

File: core/src/main/java/org/apache/calcite/sql/SqlKind.java (hunk 2 above)
Comment (vaibhavjain-dm) [reply core/src/main/java/org/apache/calcite/sql/SqlKind.java]: Done

**Pull request merged**